templates/analytics/stats.html

```diff
@@ -78,18 +78,18 @@ <h1>{{ _("Messages sent by recipient type") }}</h1>
                     </div>
                 </div>
 
-                <div class="chart-container">
-                    <h1>{{ _("Active users") }}</h1>
-                    <div id="id_number_of_users">
-                        <div class="spinner"></div>
-                    </div>
-                    <div id="users_hover_info" class="number-stat">
-                        <span id="users_hover_date"></span>
-                        <b id="users_hover_humans">{{ _("Users") }}: </b>
-                        <span id="users_hover_humans_value"></span>
-                    </div>
-                </div>
-            </div>
+      <div class="chart-container">
```

Comment:
why the spacing change?

Reply:
Thanks for pointing this, it happened while resolving a merge conflict.